### PR TITLE
Turn off debugging by using DEBUG=false

### LINKED ISSUE #2959: DEBUG=false should disable debug
Currently there's no other way to disable debug than `unset DEBUG`.

### PR DESCRIPTION
**Pull request by kke**
Fixes #2959

### CODE REVIEW

File: cli/lib/kontena_cli.rb [hunk 1]
```diff
@@ -158,11 +159,18 @@ def self.logger
...
-    logger.level = ENV["DEBUG"] ? Logger::DEBUG : Logger::INFO
+    logger.level = debug? ? Logger::DEBUG : Logger::INFO
     logger.progname = 'CLI'
     @logger = logger
   end
 
+  DEBUG_KEYWORDS = Set.new(%w(true api websocket plugin))
+
+  def self.debug?
+    DEBUG_KEYWORDS.include?(ENV['DEBUG'].to_s)
```
Comment (SpComb) [inline cli/lib/kontena_cli.rb]: Hmm... so something like `DEBUG=1 kontena ...` would have enabled debug output before, but now gets ignored... probably not worth worrying about this too much, although maybe consider special-casing `false` instead?

Comment (kke) [reply cli/lib/kontena_cli.rb]: heh, true. `== false` would've been so much simpler.

File: cli/lib/kontena_cli.rb [hunk 2]
```diff
@@ -158,11 +159,14 @@ def self.logger
...
       logger.formatter = Kontena::Cli::LogFormatter::StripColor.new
     end
-    logger.level = ENV["DEBUG"] ? Logger::DEBUG : Logger::INFO
+    logger.level = debug? ? Logger::DEBUG : Logger::INFO
     logger.progname = 'CLI'
     @logger = logger
   end
 
+  def self.debug?
+    !ENV['DEBUG'].nil? && ENV['DEBUG'].to_s != 'false'
```
Comment (SpComb) [inline cli/lib/kontena_cli.rb]: `DEBUG=` -> `ENV['DEBUG'].empty?` should also be false.

Comment (kke) [reply cli/lib/kontena_cli.rb]: true

**Pull request merged**